tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +229,14 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    #Assert
+    assert not result
```

Comment:
The test is called `test_swap_best_by_category_no_match_is_false`. We should check that result is False, not a falsey value.

```suggestion
    assert result is False
```